docs/topics/modules/approving_build_pipeline.adoc

```diff
@@ -3,34 +3,40 @@
 
 Several minutes after starting, your build pipeline is ready to be promoted to `Stage`.
 
-The build pipeline includes steps for a `Stage` build and a `Run` build. `Stage` is a production staging area to review and test changes before they finalizing them and staging the build on `Run`.
+The build pipeline includes steps for a `Stage` build and a `Run` build. `Stage` is a production staging area to review and test your application before finalizing it and staging the build on `Run`.
 
-To review the final changes in `Stage` and promote them to `Run`:
+To `Stage` your application and review:
 
 . From your Space's dashboard, click btn:[Create] from the options at the top of the screen.
 . Click btn:[Pipelines].
-. Click the icon (image:rollout_icon.png[title="Rollout"]) next to `Rollout to Stage` in the displayed pipeline. OpenShift Online provides this public URL to access the staged Quickstart project. The Vert.x HTTP Booster Quickstart produces a simple API behind a REST endpoint over HTTP with a minimalist user interface.
+. Click the icon (image:rollout_icon.png[title="Rollout"]) next to `Rollout to Stage` in the displayed pipeline. OpenShift Online provides this public URL to access the staged Quickstart Application. The Vert.x HTTP Booster Quickstart produces a simple API behind a REST endpoint over HTTP with a minimalist user interface.
++
+image::hello_john.png[Test the Application]
 +
 [WARNING]
 ====
-If the new tab displays a `Application is not available` error message (see screen capture below), the application is not yet ready. After several minutes, refresh the page and the application appears as expected.
+If the new tab displays a `Application is not available` error message (see screen capture), the application is not yet ready. After several minutes, refresh the page and the application opens as expected.
```

Comment:
Sorry, I know I said I didn't care what word we use as long as it's active but applications opening seems odd. Maybe application loads?

Reply:
I thought about this, but it could give rise to a question, the xyz menu displays> displays what?
I guess that's why IBM recommends is displayed or Opens.

Reply:
However here, loads seems to make sense, but not in the other places below.

Reply:
I think 'opening' is tied to an actual app window opening on a desktop. When the app is in a browser, it doesn't apply.